packages/saltcorn-data/models/table.ts

```diff
@@ -910,18 +910,27 @@ class Table implements AbstractTable {
     filePath: string
   ): Promise<ResultMessage> {
     let rows;
+    const state = await require("../db/state").getState();
     try {
-      const s = await getLines(filePath, 500);
-      rows = await csvtojson().fromString(s); // todo agrument type unknown
+      let lines_limit = state.getConfig("csv_lines_limit");
```

Comment:
The 500 in the previous code was not the limit to the CSV files to be passed. Instead this is the number of lines that are used to determine the types in the created table. The file is read twice, but on the first run we don't need to traverse all of it

Reply:
Yes. This is clear, Idea to add parameter - that 500 rows can be not relevant in some cases to determine types in created tables. So I tried to give more flexibility to admin for tune this process. I think that needs to change name of "csv_lines_limit"  to better name. But no idea of better name))

Reply:
I got errors when load big csv files. When 500 rows were not enough to determine types.

Reply:
@glutamate, name changed to "csv_types_detection_rows". Please review